src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -100,7 +100,7 @@
     <!-- Reference Manager capabilities -->
     <ProjectCapability Include="ReferenceManagerAssemblies" />
     <ProjectCapability Include="ReferenceManagerBrowse" />
-    <ProjectCapability Include="ReferenceManagerCOM" />
+    <ProjectCapability Include="ReferenceManagerCOM" Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(_TargetFrameworkVersionWithoutV)' >= '3.0'"/>
```

Comment:
Am I understanding this right?

Previously for a .NET Core project, we include the capability, and the SDK removed it.
Previously for a .NET Framework project, we include the capability, and the SDK does nothing.
Now for a .NET Core project, we include the capability (if v > 3), and the SDK does nothing.
Now for a .NET Framework project, we do nothing, and the SDK does nothing.

Reply:
Agreed.

Reply:
I knew I would miss something. I’ll fix that.